### ISSUE TITLE
Enterspeed should not listen on cache refresher event

### ISSUE THREAD

**Issue by bielu** [enterspeedhq/enterspeed-source-umbraco-cms#113]
Hi Guys,
relying on cache refreshers is not great as umbraco will send this notifications on following events:

- startup
- closing application
- cache repair
- publishing
- reloading cache 
I know it was done because there was no reliable way to say if something was publish with descendants, but there is change in umbraco core which will let you to determine if you need reload children or not:
https://github.com/umbraco/Umbraco-CMS/pull/13145 
:)

**Comment by jesperweber** [comment]
Hi @bielu,

Thanks for reaching out.

Yes we know the cache refresher is not the best choice, but unfortunately our only choice for now.

I didn't knew about the upcoming change, so we will definitely look into that, however as of now it only seems to be included in Umbraco 12.1 and we still need to support 9, 10 and 11. Maybe we can handle it specific for 12.1 and onwards.

**Comment by bielu** [comment]
Yeah I was more opening ticket with target for v13+ where you might need drop more things :)
